Project.toml

```diff
@@ -12,6 +12,7 @@ InfrastructureSystems = "2cd47ed4-ca9b-11e9-27f2-ab636a7671f1"
 InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 JSON2 = "2535ab7d-5cd8-5a07-80ac-9b1792aadce3"
```

Comment:
can JSON and/or JSON2 be removed now?

Reply:
Yes, thanks.  I removed them in InfrastructureSystems but not here.